src/Application/FileProcessor.php

```diff
@@ -27,6 +27,12 @@
 
 final readonly class FileProcessor
 {
+    /**
+     * @var string
+     * @see https://regex101.com/r/llm7XZ/1
+     */
+    private const OPEN_TAG_SPACED_REGEX = '/^[ \t]+<\?php/m';
```

Comment:
use `##` delimiter for consistency:

```suggestion
    private const OPEN_TAG_SPACED_REGEX = '#^[ \t]+<\?php#m';
```